libroll.py

```diff
@@ -312,6 +312,15 @@ def append_to_path(path: str):
     """
     os.environ["PATH"] = f"{os.environ['PATH']}{os.pathsep}{os.path.abspath(path)}"
 
+####################################################################################################
+
+def clone_repo(url: str, descr: str):
+    """
+    Clone a git repository
+    """
+    run(descr, f"git clone {url}")
+    print(f"Succeeded: {descr}")
+
```

Comment:
Could you extend the usage of this to other parts of the codebase where we clone? Or add an issue for it is equally good!

Reply:
Already done :)